API.md

```diff
@@ -622,27 +652,117 @@ This endpoint allows a user to delete a safe deposit box that they own
 
     + Headers
 
-            X-Vault-Token: 7f6808f1-ede3-2177-aa9d-45f507391310
+            X-Cerberus-Token: 7f6808f1-ede3-2177-aa9d-45f507391310
+            X-Cerberus-Client: MyClientName/1.0.0
 
 + Response 200
 
     + Headers
 
             X-Refresh-Token: true
 
+# Secrets
+
+## List Paths [v1/secret/{category}/{sdb-name}?list]
+
+### List Paths [GET]
+
++ Request (application/json)
+
+    + Headers
+
+        X-Cerberus-Token: 7f6808f1-ede3-2177-aa9d-45f507391310
+        X-Cerberus-Client: MyClientName/1.0.0
+
++ Response 200 (application/json)
+
+    + Body
+
+        {
+          "request_id" : "bbdd111111a2aaa3a",
+          "lease_id" : "",
+          "renewable" : false,
+          "lease_duration" : 3600,
+          "data" : {
+            "keys" : [ "path1", "path2", "path3" ]
```

Comment:
if a key ends with `/` ex: `path4/` then it is a virtual path and doesn't contain secure data.

Not sure if or how that should be incorporated into this example